python/tests/test_graphql.py

```diff
@@ -568,6 +568,241 @@ def test_rename_graph():
     server.stop()
 
 
+def test_save_as_graph():
```

Comment:
I think we want to make sure we are testing the graph here with several graph/node/edge properties to ensure that everything we use in the UI is getting copied over